classes/Learnosity/Plugin.php

```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
 		$lrn_reports_api_url = get_option('lrn_reports_api_url','https://reports-va.learnosity.com/?v1');
 		$lrn_author_api_url = get_option('lrn_author_api_url','https://authorapi-or.learnosity.com/?v1');
 		wp_enqueue_script(
-			'learnosity-items',
-			$lrn_items_api_url,
-			array(),
-			null,
-			false
+		    'learnosity-items',
+            $lrn_items_api_url,
+            array(),
+            null,
+            true
```

Comment:
Did you mean to change this `in_footer` variable to true? If so, how come?

Reply:
Yes, adding comment:
/*
         * For items and author, in_footer is set to true. This is on purpose as the information, whether
         * the read direction is right to left is set in the plugin short-code. If we loaded the scripts
         * in the header, they'd be loaded before the short-code scripts are encountered, thus we
         * wouldn't know whether to add the rtl flag in the src tag.
         */